### PR TITLE
Android scaling issue

### PR DESCRIPTION
**Pull request by sadvit**
When setting a custom font size in the Android system, an undesirable scale of the site interface in WebView occurs.

I researched that when setting the standard textZoom (100) parameter size, this undesirable effect disappears.

This can be very useful if you need to avoid the scale of content in WebView when changing the size of system fonts, or change textZoom property directly.

Example:

`
<WebView
  textZoom={100}
/>
`

### CODE REVIEW

**Comment by Titozzz** [comment]
Can you fix the merge conflicts or do you need me to do it? I'd like to get this merged

**Comment by sadvit** [comment]
@Titozzz 
Conflicts resolved, could you review please?

**Comment by Titozzz** [comment]
I wonder if setting a default value is a good idea. Cause this means it would be "breaking". I think I'd prefer this to be opt-in 🤔?

What do you think?

File: src/WebView.android.tsx [hunk 1]
```diff
@@ -283,6 +284,7 @@ class WebView extends React.Component<AndroidWebViewProps, State> {
         onLoadingProgress={this.onLoadingProgress}
         onLoadingStart={this.onLoadingStart}
         onMessage={this.onMessage}
+        textZoom={textZoom}
```
Comment (Titozzz) [inline src/WebView.android.tsx]:
```suggestion
```

File: src/WebView.android.tsx [hunk 2]
```diff
@@ -226,6 +226,7 @@ class WebView extends React.Component<AndroidWebViewProps, State> {
       source,
       style,
       nativeConfig = {},
+      textZoom,
```
Comment (Titozzz) [inline src/WebView.android.tsx]:
```suggestion
```

**Comment by sadvit** [comment]
@Titozzz 
Yes, you are right, it can break - I deleted default value

File: src/WebView.android.tsx (hunk 1 above)
Comment (Titozzz) [reply src/WebView.android.tsx]: Same

**Pull request merged**